untitled1/src/main/com/aca/homework/week7/clickable/Button.java

```diff
@@ -0,0 +1,12 @@
+package com.aca.homework.week7.clickable;
+
+public class Button extends AbstractClickable {
+
+    public Button(String name, Action action) {
+        super(name, action);
+    }
+
+    public static Button ofButton(String name, Action action) {
```

Comment:
the ofButton static factory method is not better than the constructor. so there is no need to create it